src/Nowruz/modules/userProfile/components/about/index.tsx

```diff
@@ -9,14 +12,23 @@ import { Recommendation } from './recommendaion/recommendation';
 import { Skills } from './skills';
 import { Summary } from './summary';
 import { MainInfo } from '../mainInfo';
-import { useState } from 'react';
-import { VerifyModal } from 'src/Nowruz/modules/refer/verifyModal';
 
 export const About = () => {
   const identityType = useSelector<RootState, 'users' | 'organizations'>(state => {
     return state.profile.type;
   });
   const [openVerifyModal, setOpenVerifyModal] = useState(false);
+  const [connectUrl, setConnectUrl] = useState<string>('');
+  const verifyAction = async () => {
+    const vc = await requestVerification();
+    setConnectUrl(vc.connection_url);
+    setOpenVerifyModal(true);
+  };
```

Comment:
@jeyem : I added getConnectUrl function in the 'dashboard' and 'profile' page as well, since we had this verify modal in those pages too. However I didn't applied the interval function for getting verification status. It seems somehow annoying to call this api every 5 seconds. I prefer to have a pending action which change the status to it immediately. Whenever user is approved, he can see the result by refreshing the page or we can find other solutions. I need your opinion about it please.

Reply:
@marjan-ara this is temp solution we can extend interval to 10 sec eg but best solution is we create 1 socket to push new event and refresh a part with socket trigger, but it would take time also this API right now required as our backend did not have scheduling solution so this interval should be somewhere anyway I thought that if I even put it on BE , FE need to refresh to know verification has done so it took while to develop both socket trigger and scheduling system with correct monitoring in Backend